ldap/servers/slapd/daemon.c

```diff
@@ -493,6 +499,13 @@ disk_monitoring_thread(void *nothing __attribute__((unused)))
                     if (using_auditfaillog) {
                         config_set_auditfaillog_enabled(LOGGING_ON);
                     }
+                    if (using_external_libs_debug) {
+                        if (config_set_external_libs_debug_enabled(CONFIG_EXTERNAL_LIBS_DEBUG_ENABLED,
```

Comment:
What's the intent for these calls to set the external lib debug enabled? Is it so you don't have to enable it via dse.ldif?

Reply:
It is for disk_monitoring daemon only.
It disables the logging setting if the disk threshold was crossed (and it will set it back to the previous state after the space is available once again).

Reply:
OHHHHHh I see. That's okay then :)